### PR TITLE
Test pull request

### PR DESCRIPTION
**Pull request by william-man**
This is a test request.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,8 +8,7 @@
     "start": "node backend/server.js",
     "server": "nodemon backend/server.js",
     "client": "npm start --prefix frontend",
-    "dev": "concurrently \"npm run server\" \"npm run client\"",
-    "heroku-postbuild": "NPM_CONFIG_PRODUCTION=false npm install --prefix frontend && npm run build --prefix frontend"
```
Comment (shirleymn) [inline package.json:12]: Why was this removed?

Comment (william-man) [reply package.json:12]: oh that was when i wanted to host on heroku but i changed it to somewhere else
